_develop/connect/get-started.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: learn
+permalink: /:collection/:path.html
+---
+# Get Started
+{:.no_toc}
+
+Blockstack Connect is a Javascript library for integrating your application with Stacks v2. With Connect, you get some big benefits:
+
+<!-- -  -->
+
+* TOC
+{:toc}
+
+## Installation
+
+With yarn:
+
+```bash
+yarn add @blockstack/connect
+```
+
+With npm:
+
+```bash
+npm install --save @blockstack/connect
+```
+
+## Usage
+
+### AuthOptions
+
+Every major method you'll use with `connect` requires you to pass some options, like the name and icon of your app, and what to do when authentication is finished. In practice, this means you need to define these options, and pass them to the various API methods.
+
+The exact interface you'll use [is defined as](https://github.com/blockstack/connect/blob/master/src/auth.ts#L12:L24):
+
+```typescript
+export interface AuthOptions {
+  // The URL you want the user to be redirected to after authentication.
+  redirectTo: string;
+  manifestPath?: string;
+  finished?: (payload: FinishedData) => void;
+  authOrigin?: string;
+  sendToSignIn?: boolean;
+  userSession?: UserSession;
+  appDetails: {
+    name: string;
+    icon: string;
+  };
+}
+```
+
+- `redirectTo`: The path in your app where users go after sign in.
+- `appDetails`: an optional object which includes `appName: string` and `appIcon: string`. This will speed up the process of loading your app's information during onboarding.
+- `manifestPath`: __(optional)__ - the path in your app where your manifest.json file can be found
+- `finished`: __(optional)__ - A callback that can be invoked after authentication. This prevents having to do a whole page refresh in a new tab. One argument is passed to this callback, which is an object with `userSession` included. If included, then the `redirectTo` path is ignored, and the user will be logged in automatically.
+- `authOrigin`: __(optional)__ - The URL you'd like to use for authentication. Only necessary for local development of the authenticator.
```

Comment:
where/how to get the local url+port? any docs somewhere helping with that?

Reply:
If you're developing the authenticator locally, it's localhost:8080. Docs for that are in https://github.com/blockstack/ux/blob/master/packages/app/README.md

Reply:
FYI I removed that param from the docs - since it's mostly internal, we don't need to add the mental overhead here.